docs/faking-async-methods.md

```diff
@@ -27,3 +27,19 @@ Of course, you can still configure calls to `async` methods as you would normall
 ```csharp
 A.CallTo(() => foo.Bar()).Returns(Task.FromResult("bar"));
 ```
+
+There are also convenience overloads of `Returns` and `ReturnsLazily` that let you specify a value rather
+than a task, and configure the method to return a completed task whose result is the specified value:
+
+```csharp
+A.CallTo(() => foo.Bar()).Returns("bar");
+```

```

Comment:
The convenience extension methods for `Task<T>` were not mentioned, so I added them

Reply:
Nice.